Makefile

```diff
@@ -17,3 +17,16 @@ test: ## Run the unit tests
 
 lint: ## Inpect code syntax and writing rules
 	eslint .
+
+deploy: ## Deploy website on Web Server (Need an sshname parameter for distant connection)
+	zip -r quarto.zip zipfile server.js config pages src static package.json build devServer.js
```

Comment:
Why do you need `devServer.js` on your server?

Reply:
Because I didn't find a way to build correct production site, so to keep project advancing, It deploys as in dev and a solution will be found later